LoopKit.xcodeproj/project.pbxproj

```diff
@@ -1803,101 +1807,101 @@
 		4369F090208B0D68000E3E45 /* Views */ = {
 			isa = PBXGroup;
 			children = (
-				E9086B3024B3A7270062F5C8 /* ChartTableViewCell.swift */,
```

Comment:
[nit] What caused all of these deletions and additions?

Reply:
I sorted the Group in Xcode.  I was hoping we could try to keep the folders/groups sorted to make it easier to navigate.

Reply:
I would suggest that if we are going to do this we do it in a single commit/PR dedicated to resorting all of the folders/groups globally. Doing it piecemeal along with other changes makes it pretty much impossible to see what was actually added/removed. Leave it for now, but please let's do any later sorting separate.

Reply:
Fair enough.  Thanks!